source/refined-github.css

```diff
@@ -5,12 +5,6 @@
 	--github-border-color: var(--color-border-primary, #e1e4e8);
 }
 
-/* Make tab indented code more readable on GitHub and Gist */
```

Comment:
Can we scope it just to enterprise for now, so all of us don't have to add in custom css?

Reply:
Sure, is it possible?

Reply:
Done